tests/manage/z_cluster/cluster_expansion/temp_helper_file.py

```diff
@@ -0,0 +1,165 @@
+import logging
```

Comment:
The functions in this file should be moved to the corresponding files under ocs_ci/